tictactoe.py

```diff
@@ -43,75 +43,126 @@
 # # if userX = "11":
 
 
-while True:
-    DEFAULTSPACE = "[ ]"
-    XGAMEPIECE = " X "
-    OGAMEPIECE = " O "
-    WINMESSAGE = "\n        Victory!"
-
-    gameBoard = []
+DEFAULTSPACE = "[ ]"
+XGAMEPIECE = " X "
+OGAMEPIECE = " O "
+WINMESSAGE = "\n        Victory!"
+DRAWMESSAGE = "\n          Draw!"
+
+
+gameBoard = []
+
+for _ in range(9):
+    gameBoard.append(DEFAULTSPACE)
+
+
+def displayBoard():
+    print("\n", gameBoard[0:3], "     ", [1, 2, 3],
+          "\n", gameBoard[3:6], "     ", [4, 5, 6],
+          "\n", gameBoard[6:10], "     ", [7, 8, 9],
+          "\n")
+
+
+def checkPlay(userMove, userGamePiece):
```

Comment:
another thing I just thought of is that you could probably simplify this function. Since you are passing in a string that is a number of 1-9 and the array has indices of 0-8 you could just say something like
```
gameboard[userMove-1] = userGamePiece
```
You'd just need to check if the input is a valid input first then convert it from a string to an int

So i guess this kind of goes hand in hand with the comment I wrote above this one about making a valid input check